packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -281,6 +282,13 @@ module.exports = {
     new ManifestPlugin({
       fileName: 'asset-manifest.json',
     }),
+    new SWPrecacheWebpackPlugin({
+      dontCacheBustUrlsMatching: /\.\w{8}\./,
```

Comment:
I don't believe it's immediately apparent what this is doing, we should probably add a comment explaining we're trying to match any files with a hash (as generated by webpack).

This configuration is handed to the user when they eject, so anything to help them understand what's going on better. 😄